compiler/aarch64/codegen/OMRCodeGenerator.hpp

```diff
@@ -62,6 +62,30 @@ extern TR::Instruction *loadConstant32(TR::CodeGenerator *cg, TR::Node *node, in
  */
 extern TR::Instruction *loadConstant64(TR::CodeGenerator *cg, TR::Node *node, int64_t value, TR::Register *trgReg, TR::Instruction *cursor = NULL);
 
+/**
+ * @brief Generates the immediate value depending on the double value
+ * @param[in] value : double value
+ */
+uint32_t getFPImm(double value);
```

Comment:
I propose changing the function name to `getFP8bitImm()` or something like that.
Also use the term "modified immediate constant" from the reference manual in the comment, in order to show it is a special value.